kvbc/include/block_metadata.hpp

```diff
@@ -14,25 +14,44 @@
 namespace concord {
 namespace kvbc {
 
-const char kBlockMetadataKey = 0x21;
+using concordUtils::Sliver;
+using concord::storage::blockchain::ILocalKeyValueStorageReadOnly;
+
+class IBlockMetadata {
```

Comment:
IMO for each interface or class we should add a short comment with a description of the purpose of the given component.

Reply:
done